vscode-trace-extension/package.json

```diff
@@ -124,43 +124,56 @@
     },
     "views": {
       "trace-explorer": [
+        {
+          "type": "tree",
+          "name": "Welcome",
+          "id": "welcome",
+          "when": "trace-explorer.noExperiments || !traceViewer.serverOn"
+        },
         {
           "type": "webview",
           "id": "traceExplorer.openedTracesView",
-          "name": "Opened Traces"
+          "name": "Opened Traces",
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         },
         {
           "type": "webview",
           "id": "traceExplorer.availableViews",
           "name": "Views",
-          "when": "!trace-explorer.noExperiments"
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         },
         {
           "type": "webview",
           "id": "traceExplorer.timeRangeDataView",
           "name": "Time Range Data",
-          "when": "!trace-explorer.noExperiments"
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         },
         {
           "type": "webview",
           "id": "traceExplorer.itemPropertiesView",
           "name": "Item Properties",
-          "when": "!trace-explorer.noExperiments"
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         }
       ]
     },
     "viewsWelcome": [
       {
-        "view": "traces",
-        "contents": "No traces found [learn more](https://www.eclipse.org/tracecompass/).\n[Add traces](command:nodeDependencies.addEntry)"
+        "view": "welcome",
+        "name": "Start Server",
+        "contents": "The server is offline. \n[Resume Trace Extension](command:vscode-trace-server.start-if-stopped)",
+        "when": "!traceViewer.serverOn"
       },
       {
-        "view": "views",
-        "contents": "No trace selected [learn more](https://www.eclipse.org/tracecompass/)"
+        "view": "welcome",
+        "name": "Open Trace",
+        "contents": "There are currently no opened traces. \n[Open Folder](command:openedTraces.openTraceFolder)",
+        "when": "trace-explorer.noExperiments || !traceViewer.serverOn"
       },
       {
-        "view": "item properties",
-        "contents": "No properties to show"
+        "view": "welcome",
+        "name": "Documentation",
+        "contents": "To learn more about how to use the Trace Viewer Extension [read our documentation](https://github.com/eclipse-cdt-cloud/theia-trace-extension).",
```

Comment:
This is great. Suggestion for a subsequent PR. I wonder if there is a need to make the link configurable so that adopters of the viewer can plug-in a link to their documentation.

Reply:
After the `.vsix` file is built for an extension, it can be unzipped and the original package.json file can be modified in post.  This can be done using `jq` or running a node script from the shell.  I have done this to modify the default setting for server path with the vscode-trace-server extension.

Reply:
Thanks for the suggestion!